src/reducers/importText.ts

```diff
@@ -199,30 +199,49 @@ const importText = (state: State, { path, text, lastUpdated, preventSetCursor, r
 
     const uuid = createId()
 
+    const isDestContextEmpty = getAllChildren(state, context).length === 0
+
+    /** Check if destination's parent context has more than one children. */
+    const isDestParentContextEmpty = () => getAllChildren(state, rootedParentOf(state, context)).length <= 1
+
+    const destEmpty = destThought.value === '' && isDestContextEmpty
+
+    const noNeedOfCollapse = (!destEmpty && isDestContextEmpty) || (destEmpty && isDestParentContextEmpty())
```

Comment:
I think this is the same as  `destEmpty ? isDestParentContextEmpty() : isDestContextEmpty`? If so, that might be easier to read.